### ISSUE TITLE
updates event call update event last change timestamp

### ISSUE THREAD

**Issue by dragsu** [MISP/PyMISP#876]
PyMISP Version 2.4.155.1

**Issue**
I was trying to add a local tag to an event using `update_event` call and it also updated the `last change` timestamp. Any modification to local tags should not update the `last change` timestamp. 

I also tested just by calling  `update_event` with no changes to the event and it still updates the `last change` timestamp.

**Expected outcome:**

- Changes to any local tags should not update the `last change` timestamp
-  `update_event` call without any changes to the event should not update the `last change` timestamp

**Comment by dragsu** [comment]
Have set the `__force_timestamps = True` and resubmit the request and received,

```
Event could not be saved. Event in the request not newer than the local copy.
```

**Comment by dragsu** [comment]
@Rafiot I would like to push a fix for this. Can you please add me as a contributor to the project?

**Comment by Rafiot** [comment]
You can always do a pull request if you want to contribute to the project.

Have you tried this method: https://github.com/MISP/PyMISP/blob/main/pymisp/api.py#L3443 ? It should do all you need and can be used of attributes or events.

I'll play with it to make sure the timestamp isn't updated but if it is, it is a bug we'll fix.

**Comment by Rafiot** [comment]
Yep, confirmed, you can use the `tag`method and if you pass `local=True` it won't update the timestamp.

**Comment by dragsu** [comment]
I just tested it too. It is working. Couldn't find that endpoint documented in `https://www.misp-project.org/openapi/#tag/`

**Comment by Rafiot** [comment]
Let's add @righel in the loop for that last point.

**Comment by dragsu** [comment]
@Rafiot When I try to push a branch `remote: Permission to MISP/PyMISP.git denied to dragsu.`

**Comment by Rafiot** [comment]
yes, you need to make a fork, push to your own repository and then make a pull request
